tests/lib/layer/format/mvt.test.mjs

```diff
@@ -1,167 +1,33 @@
-                            '7': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#FA5652'
-                                }
-                            },
-                            '8': {
-                                'label': 'Highest',
-                                'style': {
-                                    'fillColor': '#F0304D'
-                                }
-                            }
-                        }
-                    },
-                    'theme_2': {
-                        'title': 'theme_2',
-                        'type': 'categorized',
-                        'field': 'numeric_field',
-                        'cat': {
-                            '1': {
-                                'label': 'Lowest',
-                                'style': {
-                                    'fillColor': '#6B2E94'
-                                }
-                            },
-                            '2': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#8B44B8'
-                                }
-                            },
-                            '3': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#9B6FCD'
-                                }
-                            },
-                            '4': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#89A7D6'
-                                }
-                            },
-                            '5': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#70C1C9'
-                                }
-                            },
-                            '6': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#52C4A3'
-                                }
-                            },
-                            '7': {
-                                'label': '-',
-                                'style': {
-                                    'fillColor': '#38B77C'
-                                }
-                            },
-                            '8': {
-                                'label': 'Highest',
-                                'style': {
-                                    'fillColor': '#1FA855'
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+import mvt_layer_default from '../../../assets/layers/mvt/layer.json';
```

Comment:
Example of importing the asset. This will be applied more as I continue the re-review and structure of current tests